apps/meteor/ee/server/models/raw/LivechatUnitMonitors.ts

```diff
@@ -1,11 +1,45 @@
 import type { ILivechatMonitor } from '@rocket.chat/core-typings';
 import type { ILivechatUnitMonitorsModel } from '@rocket.chat/model-typings';
-import type { Db } from 'mongodb';
+import type { Db, DeleteResult, FindCursor, UpdateResult } from 'mongodb';
 
 import { BaseRaw } from '../../../../server/models/raw/BaseRaw';
 
+// Fix this to use the right typing
```

Comment:
I believe we can remove this file now :) https://github.com/RocketChat/Rocket.Chat/blob/chore/ee-models-to-raw/apps/meteor/ee/app/models/server/models/LivechatUnitMonitors.js#L0-L1